Content.Server/Administration/Commands/BriefCommand.cs

```diff
@@ -0,0 +1,105 @@
+using System.Linq;
+using Content.Server.GameTicking;
+using Content.Server.Players;
+using Content.Shared.Administration;
+using Content.Shared.Roles;
+using Robust.Server.Player;
+using Robust.Shared.Console;
+using Robust.Shared.Prototypes;
+
+namespace Content.Server.Administration.Commands
+{
+    [AdminCommand(AdminFlags.Admin)]
+    public sealed class BriefCommand : IConsoleCommand
+    {
+        [Dependency] private readonly IEntityManager _entities = default!;
+        [Dependency] private readonly IPrototypeManager _prototypeManager = default!;
+
+        public string Command => "brief";
+        public string Description => "Makes you a mob of choice until the command is rerun.";
+        public string Help => $"Usage: {Command} <outfit> <name> <entity>";
+
+        public void Execute(IConsoleShell shell, string argStr, string[] args)
+        {
+            var player = shell.Player as IPlayerSession;
+            if (player == null)
+            {
+                shell.WriteLine("Nah");
+                return;
+            }
+
+            var mind = player.ContentData()?.Mind;
+
+            if (mind == null)
+            {
+                shell.WriteLine("You can't spawn here!");
+                return;
+            }
+
+            if (mind.VisitingEntity != default)
```

Comment:
Doesn't this completely break if you're already visiting some entity, as it simply assumes brief is the one doing the visiting?

Reply:
Aghost does the same, what do you suggest I do to fix this?